web/packages/teleport/src/config.ts

```diff
@@ -643,6 +651,29 @@ const cfg = {
     return generatePath(cfg.routes.integrationEnroll, { type, subPage });
   },
 
+  /**
+   * Generates a route for an Integration's enrollment page when the integration is a child of an existing integration
+   *
+   * @param {string} [type] - The parent integration type (e.g. "okta", "aws-oidc")
+   * @param {string} [name] - The parent integration name (e.g. "teleport-dev", "prod")
+   * @param {string} [subType] - The child integration type (e.g. "aws-console")
+   * @param {string} [subPage] - Optional subpage within the enrollment flow
+   * @returns {string} Generated enrollment route
+   */
+  getIntegrationEnrollChildRoute(
+    type: string,
+    name: string,
+    subType: string,
+    subPage?: string
+  ) {
+    return generatePath(cfg.routes.integrationEnrollChild, {
+      type,
+      name,
+      subType,
+      subPage,
```

Comment:
should we ensure optional field like `subPage` is set to undefined if empty? i just remember there is a bug if it was empty string versus an undefined (or maybe i got that mixed up)